<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMMainWindow</class>
 <widget class="QMainWindow" name="QMMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>result output</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>768</width>
      <height>204</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontallLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Name Input</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>hit_target_factor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>endurance</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>nimble</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEdit_5"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_6"/>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEdit_7"/>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="lineEdit_8"/>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="lineEdit_13"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_9"/>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="lineEdit_15"/>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="lineEdit_10"/>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="lineEdit_14"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_11"/>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="lineEdit_12"/>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="lineEdit_16"/>
           </item>
           <item row="4" column="0">
            <widget class="QLineEdit" name="lineEdit_29"/>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_25"/>
           </item>
           <item row="4" column="2">
            <widget class="QLineEdit" name="lineEdit_28"/>
           </item>
           <item row="4" column="3">
            <widget class="QLineEdit" name="lineEdit_26"/>
           </item>
           <item row="5" column="0">
            <widget class="QLineEdit" name="lineEdit_30"/>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEdit_27"/>
           </item>
           <item row="5" column="2">
            <widget class="QLineEdit" name="lineEdit_31"/>
           </item>
           <item row="5" column="3">
            <widget class="QLineEdit" name="lineEdit_32"/>
           </item>
           <item row="6" column="0">
            <widget class="QLineEdit" name="lineEdit_22"/>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEdit_19"/>
           </item>
           <item row="6" column="2">
            <widget class="QLineEdit" name="lineEdit_20"/>
           </item>
           <item row="6" column="3">
            <widget class="QLineEdit" name="lineEdit_24"/>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>total_potence</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>position</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_13</tabstop>
  <tabstop>lineEdit_14</tabstop>
  <tabstop>lineEdit_29</tabstop>
  <tabstop>lineEdit_30</tabstop>
  <tabstop>lineEdit_22</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>lineEdit_11</tabstop>
  <tabstop>lineEdit_25</tabstop>
  <tabstop>lineEdit_27</tabstop>
  <tabstop>lineEdit_19</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>lineEdit_15</tabstop>
  <tabstop>lineEdit_12</tabstop>
  <tabstop>lineEdit_28</tabstop>
  <tabstop>lineEdit_31</tabstop>
  <tabstop>lineEdit_20</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>lineEdit_10</tabstop>
  <tabstop>lineEdit_16</tabstop>
  <tabstop>lineEdit_26</tabstop>
  <tabstop>lineEdit_32</tabstop>
  <tabstop>lineEdit_24</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
